src/toolbox.ts

```diff
@@ -555,20 +578,20 @@ export class BrushToolboxItem extends ToolboxItem {
         #toolbox div.brush span.brush-mode {
             display: flex;
         }        
-        `
+        `;
         // Create an id so this specific style tag can be referenced
-        const style_id = "brush-toolbox-item-styles"
+        const style_id = "brush-toolbox-item-styles";
 
         // Don't add the style tag if its already been added once
-        if (document.getElementById(style_id)) return
+        if (document.getElementById(style_id)) return;
 
         // Grab the document's head and create a style tag
-        const head = document.head || document.querySelector("head")
-        const style = document.createElement('style');
+        const head = document.head || document.querySelector("head");
+        const style = document.createElement("style");
```

Comment:
Might as well add a comment on this one just to make sure we're in agreement. 
Single vs double quotes is mildly controversial (see [this thread](https://stackoverflow.com/questions/25453864/what-is-the-standard-for-use-of-quotes-in-typescript)). 

`prettier` prefers double. JSON requires double. Most other strong opinions always have an exception for complex escaped strings. Some arguments claim that a single quote implies the remainder of the string doesn't have escape sequences or templating. 

I don't care either way. This is one where `prettier` has an option for it.

Reply:
I prefer double